### DISCUSSION TITLE
Provide Spending Limit or quotas at the organization level

### DISCUSSION THREAD

**Discussion by thaiknguyen** [community/community#68775] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

There are Enterprise spending limits available, but different orgs within the enterprise normally has different budgets and requirements.  It would be good to have a way to have a spending limit cap per organization.

**Comment by vishalkrishna8** [comment]
Hii @thaiknguyen , GitHub Enterprise offers spending limits primarily at the enterprise level, not per organization within the enterprise. To implement spending limits per organization, you may need a custom solution using GitHub Actions and the GitHub API.
